src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundUserAcceptanceTest.java

```diff
@@ -48,86 +50,136 @@ void tearDown() {
         System.setProperty("LAB_RESULTS_SCHEDULER_ENABLED", "false");
     }
 
-    @Test
-    void testSendEdifactIsProcessedAndPushedToGpOutboundQueue()
-        throws JMSException, IOException, InterchangeParsingException, MessageParsingException, JSONException {
+    /**
+     * The test EDIFACT message (.edifact.dat file) is sent to the MESH mailbox where the adaptor receives inbound
+     * messages. The test waits for the messages to be processed and compares the FHIR message published to the
+     * GP Outbound Queue with the expected FHIR representation of the original message sent
+     * (.[messageNumber].fhir.json file having the same name as the .dat). The json files must be numbered in the same
+     * order as the messages in the EDIFACT.
+     *
+     * E.g: testMessage.edifact.dat, testMessage.1.fhir.json, testMessage.2.fhir.json
+     */
+    @ParameterizedTest(name = "[{index}] - {0}")
+    @ArgumentsSource(SuccessArgumentsProvider.class)
+    void testEdifactIsSuccessfullyProcessedAndPushedToGpOutboundQueue(String category, TestData testData)
```

Comment:
category isn't used in the method, but its value is read by the ParameterizedTest annotation; the {0} means the first parameter in the method. 

It might be better renaming it to something else to make it more clear, something like testCategoryName or testGroup possibly?